### ISSUE TITLE
The Bactrian Questline Needs Further Refinement at the End (plus some other general problems that playing through it emphasized)

### ISSUE THREAD

**Issue by SimulatedKnave** [endless-sky/endless-sky#7826]
### Problem Description

TL;DR: unlocking the Bactrian devolves into lunacy and frustration (not because it's impossible, but because if you do it this way rather than just capping one you are intentionally making your life difficult, and the game rubs IMO your nose in that). It also shows off numerous other issues in the game which makes the whole line a disappointing experience. 

First, figuring out how to make the Deep like you requires Googling (delivering mysterious boxes leading to a twenty-five-million credit worldship is not an obvious link). So you get into it. You may, of course, risk skipping out on investigating the Deep for Foster, because you don't want to irk them. Or not help the Skadenda, for the same reason. (I didn't, but I could easily see people not risking it). But you can slowly slog through, trying to progress it.  Consciously trying to do it, it took me about fifteen hours of playtime (I admittedly was doing the Free Worlds campaign at more-or-less the same time). That seems a lot to unlock a ship. 

Oh! And I put the Free Worlds campaign on hold (i.e. I didn't join yet), because it seems like maybe good Republicans wouldn't sell a 25M city-ship to a commissioned officer of a foreign military. 

So, first problem - it takes forever, and a lot of other interesting content implies that if you do it you may hold up getting a Bactrian. If it won't, fine, but that's not clear and if it will, that's insane. Like a lot of storyline stuff in the game, figuring out getting into it involves googling rather than any real direction from the game. 

2) By the time I was blockading Beelzebub, I had a Cruiser (dozens of good Navy officers died defending that surveillance drone Katya wanted), several Falcons, and a bunch of Ospreys and Firebirds. And possibly some Bastions. In short, I had quite a bit of firepower. So I decided to capture two of the Cruisers. 

It should be noted that the Republic forces had zero apparent difficulty with the pirates, and me being there turned it into a flat out rout where everything died in seconds unless I specifically tried to drag it out. 

This, BTW, is the second playthrough I'm doing. In this one I am intentionally restricting myself to try to play through the story, rather than haring off after fancy alien tech and breaking game balance across my knee. It would be trivial for a player to come to this mission with a stronger force than I did. The shortage of lucrative things to do with that much firepower was a much bigger bottleneck than anything else. 

3) Capturing was tricky on the first one and borderline impossible on the last (because Navy and a very, very spiteful Capture RNG, unequalled before or since). On the final attempt over Farpoint, I somehow lucked into TWO Cruisers being disabled. Fair compensation for how much time it took, I thought. I shall have four Cruisers! I captured one, landed to reload Crew...only for the mission to complete and the second Cruiser to be gone when I took off again. This is galling, especially when I have discovered several uncompleted bounties that I missed where the enemy was disabled but not shot down, and I only noticed when I flew through the system again a few weeks later. 

Re those Cruisers, it would actually be extremely reasonable to have them be listed as uncapturable and just throw some text in there about orders from the Republic to shoot them down. As-is, the above experience created the impression the quest designer hated me personally. 

Also, the mission can at least only complete when every enemy is dead or captured, same as every other mission I'm aware of. Alternately, my damn bounties can complete after the ship has been sitting disabled for a week/thirty days/whatever (I'd urge this - it's surprisingly easy to miss that your fleet disabled a fighter somewhere). Pick one, but consistency is important. 

4) The Republic's complete lack of comment on me flying around with three of their Cruisers is...odd. I'm very surprised there's no "do you have a license for that ship, citizen? No? SHOOTING TIME!" event in the game. Perhaps not for the Bactrian, but certainly for Republican Naval equipment and potentially for stuff needing a Militia license. 

5) Anyway! So, I moved on. Not so long after, a kidnapping! Shock! Horror! I pursue! The pirate enemies were numerous, so I captured two of the planets as a later staging base (and because it made the Leviathan swarms less annoying). The final world tended to have three or four Leviathans defending it at once, plus reinforcements spawning constantly, so I retreated prudently. 

At some point during that, I saw a Pirate Bactrian. "OK," I said to myself, "this has officially gone on long enough. I am having one of those." I was about thirteen hours in at this point, and who knew how much longer it might be? God, they'd just started giving me a SECOND kind of mystery box mission. Much reloading for a spawn, an improved understanding of how pirate fleets spawn thanks to the Internet, and gnashing of teeth later, I had a Bactrian. I flew South, presuming that my new Bactrian would serve me well against the no-doubt numerous enemies I was about to fight on the rescue. 

6) There was a brief interlude where I gave up, flew back over to continue the mission with a hold full of nerve gas and 90% coverage from interference plating, got multiple million dollar fines in three landings, and decided to just go get the damn thing now. 

It's rather odd that they'll fine you for having stuff you shouldn't have, but won't...actually take the stuff. 

7) The reinforcements were optional. A nicetouch, I thought to myself. So I loaded up everyone I could (because surely this would be even tougher than the blockade missions), and flew off to the rescue. At this point I have the Bactrian, three Cruisers, and four Falcons (having sold numerous looted Leviathans for flipping great wodgers of cash). 

The total opposition on the way in in the final system was a Firebird. And possibly a Fury or two. We land. We take off. A Bactrian! 

Paris asks how the pirates could get such a thing. "I dunno, they have them all the damn time and you're literally in one right now," I did not respond, because that wasn't an option. I was surprisingly calm at this point, having transcended mere rage. 

Also, I believe he mentioned Hai tech at this point. I don't at any time have an option to ask him for more details about the multiple alien races he (and the various soldiers) clearly know about, even in the multiple weeks it takes us to get back home. 

8) He suggests I flee. I opt for waiting about a second, at which point it is disabled, and about another second later the Navy has (of course) blown it from the sky. 

Combat is too fast. I noticed another recent comment thread how there was a build that slowed it down and that that seemed to space things out better. I'd urge that. Frankly, I don't get much of a chance to give orders or even react in combat as it is right now. Things happen too fast, there's no ability to pause, and so few orders are given and I just kind of hope it works out for the best. Which, tbf, it usually does once I apply enough ships. I didn't even really have time to consider retreat before the damn thing was dead. Fights last seconds, or maybe a minute or two, then they're over. 

9) Screw THAT, I think, reload to before I picked the Cruisers up, and fly back in. Opposition is still a Firebird and a few tiny support ships, but this time I capture the (lovely, loaded with cool alien tech) Bactrian. 

Capturing that as my first Bactrian? Would have been awesome. As-is, it was just kind of disappointing. Because I'm not getting rid of my first Bactrian, and I don't need two, which means the neat climactic Bactrian encounter is going to be salvage. 

10) On the way out, an Aerie gets disabled. When I pop back for it (roleplaying!), the opposition is back to being swarms of Leviathans. The timely arrival of a Republican battlegroup makes capturing the place a trivial affair, and we repair the Aerie amidst the weeping widows of the Leviathan crews. 

The opposition should not get randomly weaker for the climax of the mission chain. Honestly, having the option to bring the Republic cruisers felt more like the designer trying to sneakily trick you into blowing up the Bactrian before you could capture it. 

11) I fly back. Paris' sister/wife/whatever-I-forget is dead (with her name noted twice in a slightly odd way). He tears up...then hands me a city ship license. And then talks the Bactrian, and how "it's great to fly." Which has the problem of both making him seem a psychopath, and the fact that when you're talking about real ships, not a video game, you would say 'they're'. 

Also, the idea that they're inventorying the dead two weeks later once we're back in the Deep doesn't make any sense. We'd either land on Farpoint to check, or he'd find out on the way back. Unless he just...didn't ask about his sister for two weeks. 

12) So I can finally buy a Bactrian. Which I have no need to do, having captured one in the mission, and having captured another one before that because the game kept taunting me with them. 

You know when the Deep should trust me enough to buy them? About when they trust me enough to know which of their data drops belong to their intelligence services and which belong to just private citizens. If they trust me enough to do that, they can trust me enough to sell me a ship that is all over the damn galaxy already and even in pirate hands. 

Alternately, Paris can give me the license because of the sterling work I've done and so I have one with us when we go off to rescue his sister/wife/whatever. 

I have seen a long-ago thread by one of the devs about how the Bactrian is late-game content, hard to unlock, etc. In which case, they should be much rarer than they are. Making the player work hard to get something the AI rolls around in is spiteful. 

ANYWAY. 

So: 1) Much content is hard to figure out how to access, even when you know it exists. Bactrian specifically, much OTHER content implies it'll screw up your chances of getting a Bactrian license. 

2) It seems a lot like the mission design expects a MUCH slower rate of player power growth than actually takes place. I wouldn't want stuff flat-out walled off behind a swarm of enemies, but all the content paying peanuts and vanishing in a fireball in a few seconds isn't much fun either. 

3) The Cruisers should be uncapturable (which would require the least adjustment to mission text - simply add "the Republic wants those Cruisers shot down - recapturing's too risky, and we can't let the pirates keep them"). The mission should either complete only after the Cruisers are properly destroyed (or captured, if they remain capturable), or OTHER missions can damn well start completing once you've disabled ships rather than requiring complete destruction or capture. 

4) At least some equipment that requires particular licenses should probably query you having that equipment without the corresponding license (if the relevant authorities manage to determine that you do have it). 

5) The Bactrian questline potentially bringing you into contact with capturable enemy Bactrians before you have completed is a bit of a problem. Having it do so right before the climax makes that climax an anticlimax. 

6) Illegal stuff should probably be complicated if they find you have it. Barring you fighting the enemies off. 

7) Paris' incomprehension of a Pirate Bactrian is dumb at this stage of the game. There's a very good chance the player will have seen them by now. He was, in fact, IN ONE. Also, it really does suggest the expected power curve and the actual one aren't well-related. 

8) Combat is too fast. The difference between "danger" and "death" is incredibly quick, and the relatively limited fleet controls difficult to use in the tiny timespans available for them to make a difference (this does get better a bit later, when ships are tougher. I think). 

9) Capturing that final Bactrian as your first Bactrian would actually be a fun climax to the questline (as is fleeing from it, tbf). The neat alien tech also raises questions and makes it clear there's stuff out there to find. As-is, it's an anticlimax, because if you're thirteen hours into the game you're going to be able to get a damn Bactrian by now. The only way to not have one is to not care about having one enough to get one. In which case the license is a pretty iffy reward for all this time and effort. 

10) The opposition for the actual quest should not be weaker than the random opposition I faced for scouting the quest. 

11) Paris' dialogue in the last bit of the quest needs serious work so he's not a sociopath. Also, the idea that they're inventorying the dead two weeks later once we're back in the Deep doesn't make any sense. 

12) It takes too damn long to get the license - by the time you have it, you likely have no need for it. If you want a Bactrian, capturing one is far, far, far easier than the questline. 

### Related Issue Links

Sigh. I will add this as I find them (in some cases again). 

### Desired Solution

1) As others have commented, finding quests/storylines and determining how far in you are should be easier. 
2) Not sure how player progression is currently accounted for in mission design, so not sure what the best solution is here. 
3) Uncapturable Cruisers in this mission, and generally it would be good if disabled ships autokilled themselves after a while. 
4) Where authorities find you with unlicensed equipment, some appropriate interaction should ensue. 
5) The Bactrian questline needs to give you a Bactrian earlier, and pirate Bactrians either should be locked behind the quest or Paris shouldn't have his mind blown by the very concept. 
6) Illegal equipment and goods should be seized if possible when the authorities catch you with them. While you CAN reduce all lawbreaking to a series of fines, it's an odd way to actually run a government. Nerve gas possession is illegal...but we won't actually take your nerve gas away. 
7) See 5. 
8) Combat needs to slow down and be less "spray huge quantities of firepower at enemy before they do the same to you." 
9) If possible, the final Beelzebub Bactrian should be captured by the player because that's just neat. Lean into the tendency of the game to be an orgy of boarding actions. Perhaps no liner with the passengers, "oh, take a ship with X crew capacity and plenty of marines. No reason." 
10) Make sure the enemy when you're flying into the pirate planet on the final mission is at least as strong as the enemies there the rest of the time. 
11) A slight dialogue rewrite. Hell, I can do it if necessary. 
12) Give the player the license earlier. At minimum, before they go to rescue the scientists. Ideally, back when the Deep starts trusting them with intelligence data. It's a neat ship. It's not "wait fifteen hours to unlock this when you're specifically trying to do so" neat. 

### Alternative Approaches

I think I mostly did that above.

**Comment by McloughlinGuy** [comment]
You brought up a lot of good points. I vaguely remember doing this mission, and by the time I did it I already had a whole host of ships far more powerful than the Bactrian. Combat in the game being too fast is part of a larger issue, but the missions themselves could probably benefit from some rewrites, given that many people feel the same way about the Bactrian being locked behind a quest line and so hard to discover.

**Comment by SimulatedKnave** [comment]
> 

Thanks. It was the bit where I spent time farming the damn thing, then immediately had the opportunity to capture the fancy alien one, THEN was given the license that finally broke me. 

I am now wandering through the codebase trying to find the relevant file to draft a rewrite of the minor bits. That I'm probably qualified to do.

**Comment by bene-dictator** [comment]
Sorry I gotta say this... but your playstyle is the epitome of everything this game does badly. There's nothing wrong with it because the game allows you to do it, but it makes everything so easy, so dumb, and really not fun. The campaign was one of the ways that pushed back on this min-max playstyle, but it seems to have made some things worse. 

I've tried playing a captain without any boarding at all, but because I'm so used to boarding up to an Argosy then to an Osprey and then to whatever, it's spoiled other other strategies in the early game. It really is no fun either way. 

> I captured one, landed to reload Crew...only for the mission to complete and the second Cruiser to be gone when I took off again. This is galling, especially when I have discovered several uncompleted bounties that I missed where the enemy was disabled but not shot down, and I only noticed when I flew through the system again a few weeks later.

The Cruisers despawn because they don't have the staying or nemesis personality or whatever it is. Bounty ships do because they spawn in a random system and can't be despawned or the mission wouldn't work. The Cruisers there don't because you're not meant to capture it, and the battle is predetermined. Endless Sky has no way of telling you this apart from the story, I suppose, and even then it does a bad job. 

> As others have commented, finding quests/storylines and determining how far in you are should be easier.

I must have been one of the few people who did the Bactrian missions without any help and don't care at all about farming (soulless work that destroys the game's balance), but it was a long mission chain and I did try to do things related to the Deep just for the Bactrian. I did a ton of those cube things before noticing that there were things to do with the cubes. It could be a lot shorter and there should be some guidance. On the other hand, I don't think the game should ever tell you where you are in a story, it spoils it if it's too explicit that it can tell you that you're near the end. 

> Where authorities find you with unlicensed equipment, some appropriate interaction should ensue ... Nerve gas possession is illegal...but we won't actually take your nerve gas away.

>Paris' incomprehension of a Pirate Bactrian is dumb at this stage of the game. There's a very good chance the player will have seen them by now.

> The Cruisers should be uncapturable (which would require the least adjustment to mission text - simply add "the Republic wants those Cruisers shot down - recapturing's too risky, and we can't let the pirates keep them")

> The Republic's complete lack of comment on me flying around with three of their Cruisers is...odd. I'm very surprised there's no "do you have a license for that ship, citizen? No? SHOOTING TIME!" event in the game.

Yes. Pirates should never fly Bactrians (ever, doesn't matter if you've done this campaign), otherwise Lieutenant Paris would be a busier man. I have no idea why you're able to capture Cruisers or Bactrians without penalty, if you have a ship and you're not licensed for it you should be called a pirate and get shot out of the sky by the authority that issues those licenses. That's half the reason why pirates exist, they don't have pilots' licenses to fly their ships. 

Apparently the Escape Velocity games (don't know if it's all of them, it doesn't matter) has a ton of licenses blocking purchases, that can only be unlocked through storylines. It seems like Endless Sky tried to set this up and ended up forgetting about it, and I feel like it might be too late to change it now that everyone is used to having everything available and being able to buy whatever you want. 

I don't think I've ever seen a Pirate Bactrian, and I've played a lot of the game. But I wasn't looking for them like you were, and I doubt anybody who isn't 'in the know' would have either. 

> Not sure how player progression is currently accounted for in mission design

> The reinforcements were optional. A nicetouch, I thought to myself.

> Make sure the enemy when you're flying into the pirate planet on the final mission is at least as strong as the enemies there the rest of the time.

Except for combat reputation, pretty sure it isn't. You get big ships, everything easy now. Absolutely no penalty. Why doesn't everybody in the galaxy do it? That other one should be simple, just make Beelzebub's fleet and make that fleet the only one to spawn. It probably just uses a ton of "Large Northern Pirates" fleets right now, but I haven't looked at it.   

> The Bactrian questline needs to give you a Bactrian earlier

> If possible, the final Beelzebub Bactrian should be captured by the player because that's just neat. Lean into the tendency of the game to be an orgy of boarding actions. Perhaps no liner with the passengers, "oh, take a ship with X crew capacity and plenty of marines. No reason."

End the story with capturing the Cosmic Devil for the player to keep it is a bad idea. I also don't want a Bactrian given to me or there's no point in the 25m cost if it's free. The game is stupid simple because of boarding. It's easily the most mindless way to make money. Click accept bounty mission and jump, tab until enemy ship is disabled, click click click kill crew button and get lucky, take ten minutes to land, outfit / sell ship. The last thing this game needs is an encouragement to boarding, no thank you. If you want some in-game reason, the Navy has a no boarding policy, and it'd seem hypocritical to allow the player to do it in their name. Boarding's just messed up.

**Comment by Hurleveur** [comment]
Damn this was entertaining to read.
Anyway for licenses we now can do missions triiggering with ships you dont have the license for but for navy I think it should wait until we can actually get said license trough a navy campaign.

Also the jobs are hinted at every time you do an escort mission (which you have to do like 3 times to get the bactrian) they say at the end "if u wanna help us go job board".

**Comment by SimulatedKnave** [comment]
bene-dictator: 
> Sorry I gotta say this... but your playstyle is the epitome of everything this game does badly. There's nothing wrong with it because the game allows you to do it, but it makes everything so easy, so dumb, and really not fun. The campaign was one of the ways that pushed back on this min-max playstyle, but it seems to have made some things worse.

...Yeah. The game's loaded with people who see "loot the Quarg for their stuff" as the proper way to play, and I'M the one whose playstyle is off. Sure. :P 

I'm not min-maxing. I'm simply capturing a fraction of the SWARMS of enemy ship the game throws at me, and using them to not die constantly. Y'know, like ship captains have done throughout history. Barring the Bactrian and the Cruisers I mention in this, I HAVEN'T been seeking things out to board them - just taking what I happen across. Hell, I blow up millions in fighters because I can't be bothered to board them. I don't even board Firebirds any more. 

Seriously, I'm running around with all humantech ships using normal humantech weapons, and I'm like four or five game years in. I don't think that's typical. 

> I've tried playing a captain without any boarding at all, but because I'm so used to boarding up to an Argosy then to an Osprey and then to whatever, it's spoiled other other strategies in the early game. It really is no fun either way.

Flying from star to star in a Shuttle doing low-paying deliveries is dull. There's not really any way around it. It's not helped by the extreme shortage of low-end ships. I've started a third playthrough (where I'll focus on the FW campaign), and am running into that frustration. The Heavy Shuttle is so little of an upgrade it's not really worth paying for, and after the Heavy Shuttle anything comparable is extremely expensive (1.5M for the Bounder?). Especially since most of the cargo ships are so, so slow, and the stream of enemies is near-constant. I've died a few times in the Shuttle, and it's relatively quick. I shudder to think what'd happen to me even in an unmodified Argosy. There's very little in ships to buy around the 700K mark, which is about what you'd get from picking off a few pirates or running a lot of shipping. Especially in the South of the galaxy, where the cheapest upgrade from the Shuttle is the Clipper, which is NOT cheap. 

It's not much of a choice. Especially since much of the scripted content (and the decently-paying content) expects you to fight, and if you're going be fighting it makes sense to make sure your ships can do that decently before other considerations. 

> The Cruisers despawn because they don't have the staying or nemesis personality or whatever it is. Bounty ships do because they spawn in a random system and can't be despawned or the mission wouldn't work. The Cruisers there don't because you're not meant to capture it, and the battle is predetermined. Endless Sky has no way of telling you this apart from the story, I suppose, and even then it does a bad job.

Ah, that makes sense. I mean, sort of. 

It would seem like a good idea to have disabled ships with that personality still despawn at SOME point. I genuinely came back to a system months later and discovered a bounty I'd accidentally left sitting there disabled. 

> I must have been one of the few people who did the Bactrian missions without any help and don't care at all about farming (soulless work that destroys the game's balance), but it was a long mission chain and I did try to do things related to the Deep just for the Bactrian. I did a ton of those cube things before noticing that there were things to do with the cubes. It could be a lot shorter and there should be some guidance. On the other hand, I don't think the game should ever tell you where you are in a story, it spoils it if it's too explicit that it can tell you that you're near the end.

While normally I would agree with you, finally snapping and going "I'm just going to capture a Pirate one the mission before I could have captured a Pirate one or gotten the license at the end was...EXTREMELY annoying. Some method of measuring how close I am to getting the damn thing I am working toward would be good. 

> Yes. Pirates should never fly Bactrians (ever, doesn't matter if you've done this campaign), otherwise Lieutenant Paris would be a busier man. I have no idea why you're able to capture Cruisers or Bactrians without penalty, if you have a ship and you're not licensed for it you should be called a pirate and get shot out of the sky by the authority that issues those licenses. That's half the reason why pirates exist, they don't have pilots' licenses to fly their ships.

Capturing Bactrians can sort of make sense - they ARE in the hands of multiple private owners. So maybe one of them sold it to you. Or you salvaged it. Or whatever. Pirates having them should be a big deal. 

The Cruisers...even if you DID acquire it legitimately, the Republic would presumably force you to sell it back to them or something. 

> I don't think I've ever seen a Pirate Bactrian, and I've played a lot of the game. But I wasn't looking for them like you were, and I doubt anybody who isn't 'in the know' would have either.

I mean, I wasn't looking THAT hard. They pop up a lot in the North, and I think one tried to attack me on one of the convoy runs to Tarazed (i.e. surprisingly early in the game). 

> Except for combat reputation, pretty sure it isn't. You get big ships, everything easy now. Absolutely no penalty. Why doesn't everybody in the galaxy do it? That other one should be simple, just make Beelzebub's fleet and make that fleet the only one to spawn. It probably just uses a ton of "Large Northern Pirates" fleets right now, but I haven't looked at it.

If I were guessing, I suspect the only time the mission specifies the opposition is when you're actually doing the final mission - i.e. when you scout the place, you get what your combat rep deserves. When you're in the mission, you get what is designed. 

Also, everyone in the galaxy DOES do it. Where else do the pirates get their endless stream of Leviathans etc? 

> End the story with capturing the Cosmic Devil for the player to keep it is a bad idea. I also don't want a Bactrian given to me or there's no point in the 25m cost if it's free. The game is stupid simple because of boarding. It's easily the most mindless way to make money. Click accept bounty mission and jump, tab until enemy ship is disabled, click click click kill crew button and get lucky, take ten minutes to land, outfit / sell ship. The last thing this game needs is an encouragement to boarding, no thank you. If you want some in-game reason, the Navy has a no boarding policy, and it'd seem hypocritical to allow the player to do it in their name. Boarding's just messed up.

Any ship in the game is free if you have a large enough crew capacity and a powerful enough fleet. By the time the damn Bactrian is actually UNLOCKED, 25M is trivial. I made more than that just scouting the damn planet out by capturing a few of the Leviathans that continually attacked me. It would have been harder to NOT make the money. 

Also, the navy's no-boarding policy is insane - it wastes materiel and removes any possibility of taking prisoners. While there are good game design reasons for it, as an actual policy it'd be insane. I'm not saying it should be changed, just it shouldn't be stated as an actual in-game-world policy IMO. 

The problem is not boarding as such. Boarding is fundamental to any game about shipping. It is, for that matter, fundamental to real life shipping. The problem is there is absolutely nothing in the game to spend money on OTHER than more ships and more equipment for those ships. There's no taxes. There's no maintenance costs. Your crew are happy with their 100 credits a day, no matter how many of them were killed and how expensive that ship you just captured is. So any captured ship is free money, in at least six figures. 

Maintenance, at least, is a simple solution: the player pays a percentage of the depreciated cost of their equipment each month. So as your equipment ages, it costs more to maintain. Actually incentivizes buying new stuff if the price is right, though getting the pricing right is potentially trickier. 

Increasing crew pay is...a bit trickier. One of the major points of the game is just how cheap labour is in this world. Honestly, I think 100 credits a day is probably about right. I suppose you could double it reasonably enough, but much more than that feels too high given that life is clearly pretty damn cheap. 

Prize money is definitely a thing that should exist, but is probably complicated to implement (only thing I can think of off  the top of my head is that ships count as prizes until you get to a world with a shipyard (or at least an outfitter) at which point you are expected to either sell the thing and give a percentage to your crew, or pay to keep it. Or just claim there's a prize agent from the bank in every port and do it that way. Actually, that's EXACTLY how it should work - next time you land after you capture a ship, it's prize division time with the local bank prize agent. Makes the bank relevant forever, and makes buying ships much more common. 

Taxes are easy enough. Though, really, given how hard depreciation already hits, adding a significant tax is getting a bit much. Though different tax rates WOULD encourage people selling in different places. 

Alternately, just increase mission payouts so much that it's a competitive alternative to boarding payout wise, and limit the spawn rate of enemies so the constant stream of enemy ships disappears and there is less need for a large fleet (and less opportunity to create one from an endless stream of enemy capital ships). Seriously, the alert alarm goes off every ten seconds.

**Comment by SimulatedKnave** [comment]
hurleveur: 
> Damn this was entertaining to read. Anyway for licenses we now can do missions triiggering with ships you dont have the license for but for navy I think it should wait until we can actually get said license trough a navy campaign.
> 
> Also the jobs are hinted at every time you do an escort mission (which you have to do like 3 times to get the bactrian) they say at the end "if u wanna help us go job board".

Thanks. lol

Fair point re the waiting til you can get things legitimately before locking them up. 

I'm not clear on how one gets to unlocking that, though - I thought one had to run enough Deep missions that they liked you in the first place to GIVE you the escort missions. If that's not the case, no worries.

**Comment by Hurleveur** [comment]
You should know: all balancing of the game is done expecting you to only have one ship, the strongest available at the time. At that point that would be a leviathan/falcon.
That's smth that'll get improved in the future with different difficulties etc though.

**Comment by bene-dictator** [comment]
> ...Yeah. The game's loaded with people who see "loot the Quarg for their stuff" as the proper way to play

That's what I meant. And I said you weren't wrong, it's just how the game works. 

> Also, everyone in the galaxy DOES do it. Where else do the pirates get their endless stream of Leviathans etc?

Oh yeah, I know that. But we never see that in-game, it's all off screen. Some of it happens through planet raids, which are also not really seen but you fight them. 

> finally snapping and going "I'm just going to capture a Pirate one the mission before I could have captured a Pirate one or gotten the license at the end was...EXTREMELY annoying. Some method of measuring how close I am to getting the damn thing I am working toward would be good.

I agree, but an easier way to do it would be to make the mission sequences shorter, because it is stupid long right now, or it has to be done very vaguely. You never get an explicit number or percentage, just a status update. But definitely, you don't want to get lost in the story or have what happened to you happen to everyone. 

> Any ship in the game is free if you have a large enough crew capacity and a powerful enough fleet. By the time the damn Bactrian is actually UNLOCKED, 25M is trivial.

> The problem is not boarding as such. Boarding is fundamental to any game about shipping. It is, for that matter, fundamental to real life shipping. The problem is there is absolutely nothing in the game to spend money on OTHER than more ships and more equipment for those ships. There's no taxes. There's no maintenance costs. Your crew are happy with their 100 credits a day, no matter how many of them were killed and how expensive that ship you just captured is. So any captured ship is free money, in at least six figures.

Yes, but boarding still sucks, because it makes every other strategy in the mid-game bad. And mid-game is most of Endless Sky. The end game is pretty much capturing a good ship to use. Again, why isn't every trader just a privateer looking for pirates to board? Because they're not soulless bastards like us sending crew to their death without penalties? I don't see a good reason for having no boarding penalties that aren't bad luck and a restart. Boarding is too easy and too profitable, and it's not believable, that is why it is bad. 

I like your ideas but I think other people with the power to stick that into the game have different ideas. I'd focus on developing the mechanics of the game before the story and the fluff with new aliens and stuff, but I've never been able to get motivated with code. I can only really help the stories improve right now. That's fine, and things might change. There are always forks and plugins. 

For example:
> Maintenance, at least, is a simple solution: the player pays a percentage of the depreciated cost of their equipment each month. So as your equipment ages, it costs more to maintain. Actually incentivizes buying new stuff if the price is right, though getting the pricing right is potentially trickier.

This was talked about here: #2219, in a PR here: #852, and there was some other issue about it too that I can't find, but they were shot down by Michael for adding more complexity without adding anything interesting. I don't agree, and MZ isn't around to say no to it anymore, but I don't think there's been anything new that's happened about this. I think it would at least make credits more interesting, because right now credits are meaningless as soon as you pay off your mortgage. 

> Also, the navy's no-boarding policy is insane - it wastes materiel and removes any possibility of taking prisoners. While there are good game design reasons for it, as an actual policy it'd be insane. I'm not saying it should be changed, just it shouldn't be stated as an actual in-game-world policy IMO.

It's mentioned in the Hai Revealed story. If you're not on the continuous version, you won't know about it. It's true that the boarding policy is pretty stupid, but I don't know what ethics looks like in 3000 years, and I'm not involved in the story writing enough to explain it. 

I really do agree with most of the things you say, though.

**Comment by Hurleveur** [comment]
The mission could just have a shortcut, where if instead of obeying your orders you go to X system you can catch Belzebuth and cut it short there, if you know what you're doing.

**Comment by SimulatedKnave** [comment]
> I agree, but an easier way to do it would be to make the mission sequences shorter, because it is stupid long right now, or it has to be done very vaguely. You never get an explicit number or percentage, just a status update. But definitely, you don't want to get lost in the story or have what happened to you happen to everyone.

V sensible. 

> Yes, but boarding still sucks, because it makes every other strategy in the mid-game bad. And mid-game is most of Endless Sky. The end game is pretty much capturing a good ship to use. Again, why isn't every trader just a privateer looking for pirates to board? Because they're not soulless bastards like us sending crew to their death without penalties? I don't see a good reason for having no boarding penalties that aren't bad luck and a restart. Boarding is too easy and too profitable, and it's not believable, that is why it is bad.

I mean, I assume they do it offscreen, because if the player is annoyed by the Navy blowing up their captures the player attitude toward random civilians stealing their prizes would be something like apoplexy. And coding boarding AI is probably tricky. 

But IRL, that's pretty much how privateering worked - you were one when it was convenient, and not when it wasn't. 

Also, IRL it was slowed down by the issue of having to CATCH the enemy ship. That's not an issue here. But I guarantee you anyone who beat a pirate attack kept the damn ship that attacked them. 

> This was talked about here: #2219, in a PR here: #852, and there was some other issue about it too that I can't find, but they were shot down by Michael for adding more complexity without adding anything interesting. I don't agree, and MZ isn't around to say no to it anymore, but I don't think there's been anything new that's happened about this. I think it would at least make credits more interesting, because right now credits are meaningless as soon as you pay off your mortgage.

It worries me how right that guy is. Also, maintenance makes depreciation relevant as opposed to just a screw-you to the player. 

Given how there seem to be multiple projects adding used ships, I'd hope there'd be some interest in maintenance and prize money etc. Coding maintenance in seems like it should be trivial (despite someone in one of those threads claiming it wouldn't be) - you'd just take the "deduct money for the crew" and "deduct money for mortgages" code, throw in the value of the player's fleet instead, and divide by the right number (where the right number is whatever makes sense). I'm prepared to believe it's more involved than that, but I have a hard time seeing it as insurmountably complicated if depreciation was worth adding. 

Coding prize money is obviously much more involved, but I think it's probably necessary to keep the economy even vaguely relevant. 

And, frankly, the profitability of other stuff needs to improve to compensate. Blowing up marauders quickly scales to like 300K in payouts. Meanwhile, they'll pay me 20K to take some passengers somewhere. Not exactly comparable. 

> It's mentioned in the Hai Revealed story. If you're not on the continuous version, you won't know about it. It's true that the boarding policy is pretty stupid, but I don't know what ethics looks like in 3000 years, and I'm not involved in the story writing enough to explain it.

It turns the Navy from dedicated humanitarians (who happen to be implemented as a bit into watching things blow up) into psychopathic vandals with a near-infinite bodycount, so I'd say "pretty stupid" is an understatement.

**Comment by Zitchas** [comment]
I'm not super thrilled with the idea of watching my ship deteriorate around me. Plenty of games have "your gear wears out and you need to repair it" mechanics, and they range from minor annoyance to outright irritation (depending on how easy it is to repair and/or  prevent degradation). They never really add anything useful so far as I can tell.

**Comment by SimulatedKnave** [comment]
Zitchas: 
I mean, you're already watching your ship deteriorate around you. My Clipper's a month old and it's apparently worth 75% of what it was when I bought it. That seems improbable. At least if maintenance costs increased on older equipment it'd make some sense that depreciation is a thing. Otherwise it has no point other than to keep the value of the player's salvage artificially low and making it harder for the player to resell their barely-used starter ship. If there's no point to the mechanic other than that, remove it and make the early game easier for people NOT salvaging. 

I would expect maintenance costs to remain relatively low, even at full depreciation. I'd also expect depreciation to be slower. MUCH slower - maybe 10-20% a year, at most. Stuff should actually hold its value (though pirate and salvaged equipment can absolutely be fully depreciated. It's salvage. In a truly advanced implementation I'd expect depreciation to increase on captures as a function of how much effort it took to capture it, and depreciation levels to be different from government to government, but that's getting a bit excessive). 

And this IS adding something useful: it adds somewhere for all the money to go, and makes depreciation something other than a penalty to the player's ability to sell stuff. A penalty which doesn't slow down salvaging much, but sure screws over simple traders, at least at its current rates.

**Comment by SimulatedKnave** [comment]
...Having looked through the source code for the Bank and for the Account, maintenance is already there. 

So...yeah. 

Maybe it should be implemented?

**Comment by bene-dictator** [comment]
Yeah, that was a PR for people who wanted to make forks to do this sort of stuff. You could make a fork or plugin of this, I don't know how well a PR would stand up now but you can try. At the very least, I'd like to see it.

**Comment by Hurleveur** [comment]
Maintenance is only for a set amount, for a ship or an outfit.
If we want to make all ships make you pay an amount depending on depreciation that would require some (very small) coding to work. I'm down for this as well honestly.

**Comment by SimulatedKnave** [comment]
I'll give it a go. I can certainly draft something, but my C++ coding skills are basically zero. I've found the relevant bits for depreciation and crew salary, and THINK I'm in the right spot for maintenance, so it's just a question of figuring out what to do with it. Actually adding viewing maintenance costs for a particular ship to the interface is 100% beyond me, but I think I can at least implement it happening. 

I'll try and draft something up. Will advise once I have done so.

**Comment by Zitchas** [comment]
Thought: What is depreciation actually supposed to do?

Right now, it penalizes players for holding onto ships any longer than necessary. It pushes them to sell and upgrade ASAP because that's how they'll prevent loosing too much of their investment.

What do we *want* to have happen? Perhaps discourage fleets, or general collecting of large numbers of ships? Act as a nerf to capturing?

A better implementation of depreciation might be to have depreciation only affect non-flag ships. So, flagship doesn't depreciate, but everything else does. Then people wouldn't feel pressured to sell their flagship and upgrade ASAP, but *would* be pressured to sell everything else. I'm not sure what the lore would be for that, though.

**Comment by SimulatedKnave** [comment]
A fair question, Zitchas. 

I looked to try and see why it was implemented, and can't find why. It really does seem to mostly just penalise capturing, and encourage upgrading anything you have bought before it depreciates too badly. That second seems very unfortunate. The first is reasonable enough, but it doesn't achieve that because capturing's so damn easy given the swarm of enemy ships the game throws at you. 

Honestly, I'd support removing it over leaving it completely unchanged.

**Comment by johndh** [comment]
The brokenness of boarding is probably the most common discussion on this tracker. Despite being the most successful strategy, it's not challenging in any interesting way, just frustrating.
I like the prize money idea. I recently proposed the idea of having to substantially repair captured ships and outfits before using them #7774 , which would serve the same purpose as depreciation as it relates to nerfing capture profits, in a way that (imo) makes more sense. Prize money could do a similar thing, while also justifying why your crew are so willing to be massacred for $100/day. The current system just makes it four times more rewarding to keep the ship than to sell it.
Regarding the navy policy, the problem is that there are only two options for dealing with an enemy ship, and both require murdering everyone on it. It would only make sense if there were some way to capture a crew alive #5769 , so they're not just committing overt war crimes on every enemy they see by executing them in cold blood.

**Comment by SimulatedKnave** [comment]
> The brokenness of boarding is probably the most common discussion on this tracker. Despite being the most successful strategy, it's not challenging in any interesting way, just frustrating. I like the prize money idea. I recently proposed the idea of having to substantially repair captured ships and outfits before using them #7774 , which would serve the same purpose as depreciation as it relates to nerfing capture profits, in a way that (imo) makes more sense. Prize money could do a similar thing, while also justifying why your crew are so willing to be massacred for $100/day. The current system just makes it four times more rewarding to keep the ship than to sell it. 

I'd concur that repairing probably makes more sense than depreciation as it is currently implemented, especially given a just-captured ship is extremely vulnerable in any case. I suspect we need to be wary of the possibility of ships being recaptured after the AI boards them (I believe that used to be possible?), but beyond that either or both would seem wise. 

Also, I am increasing crew salary in anything I mod to $200. Still peanuts to the player, but a hell of a lot more to the crew (especially when you consider that their food presumably comes out of that). 

> Regarding the navy policy, the problem is that there are only two options for dealing with an enemy ship, and both require murdering everyone on it. It would only make sense if there were some way to capture a crew alive #5769 , so they're not just committing overt war crimes on every enemy they see by executing them in cold blood.

I'd assume that the boarding process actually does involve some prisoner-taking, it's just abstracted out.

**Comment by Hurleveur** [comment]
Just make depreciation twice as slow and the urge to sell wont be as big.
I think that the extra maintenance makes sense as a nerf to boarding and huge fleets. Only question is how much of the base ship cost per day when on 25% depreciation for instance.
Also I may have underestimated how hard it is to do this, it should probably come in the aame way the salaries do but based on ship cost and depreciation. Basing it on that its easy to show it in the map the same way tributes salaries etc are showed.

**Comment by Zitchas** [comment]
Making depreciation twice as slow.... for the flagships. Perhaps cap the depreciation for the flagship, too. 75% would be a good (if arbitrary) starting point. So ships do lose *some* value over time, but the flagship gets the most care and attention, and thus doesn't lose value past a certain point.

**Comment by Zitchas** [comment]
I haven't settled on lore for why that might be the case, but the first thing that comes to mind is "Who is going to take better care of a ship? The person who earned and then spent several million credits to get it and now captains it personally; or the nameless technician who is paid 100/day to make sure it gets where it is going and follow along in the wake of the owner who is off flying their own ship up ahead?"

Case in point, who takes better care of their truck? The owner-drivers who own their own vehicle? Or the drivers for big multinational companies who just show up to work and get assigned a vehicle, which is *usually* the same one, but not always? While there are, admitedly, some people who run their stuff into the ground, generally speaking the owner is likely to take better care of their own vehicle than a random driver who was merely appointed to drive that particular vehicle today.

**Comment by Hurleveur** [comment]
Thats why I made the maintenance cost inexistant for it, you take care of it yourself, meaning you don't pay extra. If we make it go slower for all flagships then it makes ships that depreciate slower less unique.

**Comment by Zitchas** [comment]
In terms of complexity, I recall someone once suggesting that instead of depreciation being time based, it should instead be based on hull damage taken. I don't recall numbers, but for example, for every 1% of it's total hull strength taken in hull damage, it loses 1% of its value. So if a ship gets pounded into a pulp and disabled, it's now lost ~60% of its value. Who is going to pay anything close to full price for a ship that's a giant mass of repair plating and welds? On the other hand, if you can keep your ship hull-damage-free for ten years, well, that seems indicative of a fair bit of care and attention, which would make a high resale value appropriate. 

It makes sense, although it seemed a bit overly complex and I'm not 100% in favor of it; but it *is* an interesting concept.

**Comment by yafflehk** [comment]
Tie ship depreciation to combat score? If you're a bloodthirsty maniac your ships must get used hard and take a lot of damage so they depreciate fast. Conversely if you're a cautious merchant captain who stays in Hai space your ships will hold their value way longer.

**Comment by Hurleveur** [comment]
Trading is already too good though idk about buffing it more honestly
The fact that your captured ship will instantly cost full upkeep prices whereas it'll take years for the bought one to already is smth.

**Comment by LiteraryWho** [comment]
>  Prize money could do a similar thing, while also justifying why your crew are so willing to be massacred for $100/day. 

Total aside, but a hundred credits gets you about a ton of food in the right places.  Your crew are actually getting paid extremely well.

**Comment by LiteraryWho** [comment]
On topic, I think maybe in game rumors in the form of cultural missions might be a good way to point a player towards how to get the Bactrian/other exotic ships.

Heck, I could probably write something for that...
